### PR TITLE
Aug Summit Content on the monthly update 

### PR DESCRIPTION
**Pull request by stevetrask**

### CODE REVIEW

File: resources/monthly-update.md [hunk 1]
```diff
@@ -8,6 +8,115 @@ description: See the latest updates from the previous and upcoming monthly membe
...
+
+<!-- <div class="videoWrapper mb-10">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/M-XiDO1VqT4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div> -->
+
+<div class="bg-gray-200 rounded-lg px-6 py-4 text-center">
+  <p><a href="/membership/" data-ga-category="links" data-ga-action="internal link clicks" data-ga-label="membership - monthly update">Become a FinOps Foundation member</a> to join the FinOps summit on August 19th</p>
+</div>
+
+This month it happened again, we have too many amazing speakers and content areas to cover! We hope you join hundreds of your peers at our session on Aug 19th, with over a dozen speakers across three topics:
```
Comment (andrewnhem) [inline resources/monthly-update.md:21]:
```suggestion
It happened again this month: we have too many amazing speakers and content areas to cover! We hope you join hundreds of your peers at our session on Aug 19th, with over a dozen speakers across three topics:
```

Comment (andrewnhem) [reply resources/monthly-update.md:21]: @stevetrask  Also a sooner mention of Intl FinOps Day? Seems buried below.

File: resources/monthly-update.md [hunk 2]
```diff
@@ -8,6 +8,115 @@ description: See the latest updates from the previous and upcoming monthly membe
...
+
+### Keynotes:
+### #1 Dealing with Data Transfer
+
+Based on a lively [conversation in Slack](https://finopsfoundation.slack.com/archives/CNJPGNMMJ/p1627071180232200) started by Josh Bauman (EA), we’ve assembled a panel of FinOps experts from heavy streamers to discuss the ins/outs of data transfer cost management: Anders Hagman Spotify, Teresa Meade Electronic Arts, and others.
+
+### #2 Adopting FinOps: Getting Organizational Buy-In
+
+Chairs Ashley Hromatko and Tech Lead Mike Eisenstein are back to present an updated executive buy-in deck for building a FinOps practice that the working group has been tackling. They will also unveil the new FinOps Framework poster that gives a one-page overview of the core tenants of the practice with Vas Markanastasakis. Hear from Melvin Brown (US Gov OPM) and Natalie Daley (HSBC) on their stories and follow-along in [#wg-adopting-finops](https://finopsfoundation.slack.com/archives/C026T4HQK09).
```
Comment (andrewnhem) [inline resources/monthly-update.md:30]:
```suggestion
Chair Member Ashley Hromatko and Tech Lead Mike Eisenstein are back to present an updated executive buy-in deck for building a FinOps practice that the working group has been tackling. They will also unveil the new FinOps Framework poster that gives a one-page overview of the core tenants of the practice with Vas Markanastasakis. Also hear from Melvin Brown (US Gov OPM) and Natalie Daley (HSBC) on their stories and follow-along in [#wg-adopting-finops](https://finopsfoundation.slack.com/archives/C026T4HQK09).
```

File: resources/monthly-update.md [hunk 3]
```diff
@@ -8,6 +8,115 @@ description: See the latest updates from the previous and upcoming monthly membe
...
+
+### #3 Waste Reduction: Mountains vs Molehills
+
+Borrowing from Joseph Daly’s post on the topic, we are opening up a conversation about how to tackle and prioritize the little tiny inefficiencies of cloud (molehills) vs. the optimization opportunities that can transform value (mountains). Panel includes Andrew Brixton (Citrix), Melissa Lorton and Michael Kerns from Virtasant, Scott Meyer (Spotify), Teresa Meade (EA) sharing their tales on tackling waste. Check out slack threads in [#aws](https://finopsfoundation.slack.com/archives/CNJPGNMMJ/p1627552385245600) and [#azure](https://finopsfoundation.slack.com/archives/CNJPJ3YDA/p1627552311081500) on this topic.
```
Comment (andrewnhem) [inline resources/monthly-update.md:34]:
```suggestion
Borrowing from Joseph Daly’s post on the topic, we're opening up a conversation about how to tackle and prioritize the tiny inefficiencies of cloud (molehills) vs. the optimization opportunities that can transform value (mountains). Panelists include Andrew Brixton (Citrix), Melissa Lorton and Michael Kerns (Virtasant), Scott Meyer (Spotify), and Teresa Meade (EA) sharing their tales on tackling waste. Check out slack threads in [#aws](https://finopsfoundation.slack.com/archives/CNJPGNMMJ/p1627552385245600) and [#azure](https://finopsfoundation.slack.com/archives/CNJPJ3YDA/p1627552311081500) on this topic.
```

File: resources/monthly-update.md [hunk 4]
```diff
@@ -8,6 +8,115 @@ description: See the latest updates from the previous and upcoming monthly membe
...
+
+The event is free to approved FinOps Foundation members - [join here!](https://www.finops.org/membership/) and will also feature breakout rooms on each topic as well as updates from our various working groups and Technical Advisory Council. There will be breakouts sessions during the last 30 minutes.
```
Comment (andrewnhem) [inline resources/monthly-update.md:36]:
```suggestion
The event is free to approved FinOps Foundation members - [join here!](https://www.finops.org/membership/) It'll  also feature breakout rooms on each topic as well as updates from our various Working Groups and Technical Advisory Council. There will be breakouts sessions during the last 30 minutes.
```

**Comment by jrstorment** [comment]
Please pick a consistent way to list company names and include them for everyone. I've been leaning toward "Anders Hagman (Spotify)". Also, please include Natalie Daley + Melvin Brown in the speaker list as they will be sharing stories on Adopting. Also, please confirm discounts and links with Stacy.

**Review by andrewnhem — APPROVED** [review]

**Pull request merged**